nb/api/Spiral.nb

```diff
@@ -0,0 +1,37 @@
+md`
+[index](#https://raw.githubusercontent.com/jsxcad/JSxCAD/master/nb/api/index.nb)
+### Spiral()
+Parameter|Default|Type
+---|---|---
+op|Point|Function or shape to build from.
```

Comment:
The documentation on these is really nice :)

Reply:
Yeah, moving it to the table form helped a lot, since they're mostly not positional arguments now.